client-src/elements/chromedash-feature-detail.js

```diff
@@ -29,6 +30,7 @@ class ChromedashFeatureDetail extends LitElement {
     this.canEdit = false;
     this.feature = {};
     this.gates = [];
+    this.isAddingStage = false;
```

Comment:
I don't see where this is used.

Reply:
This is a ghost from a previous implementation attempt that I missed removing. Thanks for catching!